testing/jormungandr-automation/src/testing/benchmark/sync/measure.rs

```diff
@@ -109,34 +109,29 @@ pub fn measure_and_log_sync_time<A: SyncNode + ?Sized>(
     let interval: u32 = report_node_stats_interval.into();
 
     while !benchmark.timeout_exceeded() {
-        let block_heights: Vec<u32> = nodes
+        let tips = nodes
             .iter()
             .map(|node| {
                 if report_node_stats_counter >= interval {
                     node.log_stats();
                 }
-                node.last_block_height()
+                node.tip()
             })
-            .collect();
+            .collect::<Vec<_>>();
 
         if report_node_stats_counter >= interval {
-            println!(
-                "Measuring sync time... current block heights: {:?}",
-                block_heights
-            );
+            println!("Measuring sync time... current block tips: {:?}", tips);
```

Comment:
Maybe implement `Display` trait implementation for `tips` as we display it via `println` for the user's convenience, not for the debug purposes.